src/core/common/countQuery.ts

```diff
@@ -0,0 +1,9 @@
+export interface CountQueryArgs {
+  id?: string;
+  url?: string;
+  notext?: boolean;
+}
+
+export function createCountQueryRef(args: CountQueryArgs) {
+  return btoa(`${JSON.stringify(!!args.notext)};${args.id || args.url}`);
```

Comment:
This appears to be repeated from `src/core/client/count/index.ts` as well.

Also, while your solution is more compact, wouldn't using `btoa(JSON.stringify(args))` be more readable?

Reply:
Oh the duplicate is actually not used, will remove this function. Otherwise it makes sense to simplify it to your version :-)

Reply:
Oh actually I remember why I did it that way... it was to save characters 🤔

Reply:
I would prefer the current version that saves quite a few characters, maybe just a comment to make it clear what it does?